docs/setup.rst

```diff
@@ -5,6 +5,30 @@ Installation
 
 This guide will bring you through the steps of creating a local Semester.ly server and development environment. It will walk through the setup of the core ecosystems we work within: Django/Python and React/Node/JS. It will additionally require the setup of a PostgreSQL database.
 
+Setting up Visual Studio Code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We recommend using `Visual Studio Code <https://code.visualstudio.com/>`_
+(VSCode) for its integration with WSL 2, Docker, and the Postgres database. 
+This section assumes you will be using Visual Studio Code for development with 
+Semester.ly.
+
+1. If you are on Windows OS, see the following guide on
```

Comment:
Add some marker/section indicating very clearly this is only necessary for Windows and/or Ubuntu (not for MacOS)